### ISSUE TITLE
What is the wrong of test results in the official GANimation project?

### ISSUE THREAD

**Issue by ewrfcas** [donydchen/ganimation_replicate#11]

**Comment by donydchen** [comment]
There are lots of possibilities for the wrong test results of deep network, but perhaps no certainty. Most likely it is related to the way of training, e.g. learning rate, decay strategy, etc. Since the author of the official implementation has decided not to release the pre-trained model, others may not be easy to name certain reasons for the failure of test case.
